modules/panorama/main.tf

```diff
@@ -94,7 +95,10 @@ resource "azurerm_managed_disk" "this" {
   storage_account_type = "Standard_LRS"
   create_option        = "Empty"
   disk_size_gb         = lookup(each.value, "size", "2048")
-  zones                = [lookup(each.value, "zone", "")]
+  zones = try(
+    [each.value.zone],
+    var.enable_zones && var.avzone != null && var.avzone != "" ? [var.avzone] : null
+  )
```

Comment:
Before, omitting zone on logging_disks always resulted in
Error: expected "zones.0" to not be an empty string and thus it
had effectively no working default value.

Now the same input will default to the value of input avzone,
so that the disk is placed together with the owning virtual machine
and not susceptible to accidental zone separations.